addon/styles/addon.scss

```diff
@@ -46,5 +54,11 @@
   .animate {
     transition: transform 0.2s ease;
   }
+}
 
+.frost-sort-container {
```

Comment:
Since the style is prefixed by `.frost-sort0-` it's acceptable to list it outside of the `.frost-sort` block.  Generally what I would do though is

```
.frost-sort {
  &-container {
```

Which will result in a `.frost-sort-container` style in the final CSS